### ISSUE TITLE
pamac 6.0.1 refresh DB bug

### ISSUE THREAD

**Issue by elishl** [manjaro/pamac#298]
Please consider the following test case:

Switch to testing branch using pacman-mirrors.
Install a program (for example, dropbox).
Switch to unstable branch using pacman-mirrors where an update for dropbox is available.
Pamac will show that an update is available, all good.
Switch back to testing branch using pacman-mirrors.
Pamac will still inform the user about an update to dropbox (even though pacman -Syu outputs "nothing to do").

In short, pamac "sees" if an update is available but doesn't remove it in case of branch reversion.

Thanks for the hard work,
Eli.

**Comment by philmmanjaro** [comment]
Did you do a DB-refresh after switching branches?

**Comment by elishl** [comment]
Of course.
Pacman -Syyu after every branch change.

**Comment by philmmanjaro** [comment]
I meant within pamac.

**Comment by elishl** [comment]
Yep.
Done that through Pamac as well.
Still the same problem.

**Comment by guinux** [comment]
Did you restart pamac after switching branches?

**Comment by guinux** [comment]
Should be fixed with 34130bb

**Comment by elishl** [comment]
Thank you very much.
Will test as soon as the new version hits the repos.

**Comment by philmmanjaro** [comment]
@elishl: you may test it with **pamac 6.0.1-2**.

**Comment by elishl** [comment]
@guinux and @philmmanjaro ,
Works very well.
Thank you for your hard work.
